yarn.lock

```diff
@@ -1123,15 +1172,15 @@ vue@^3.2.25:
     "@vue/shared" "3.3.13"
 
 vue@^3.3.9:
```

Comment:
which command did you run ? `yarn upgrade` right ?

maybe change the the `package.json` as well to reflect the change ? `^3.4.19`

https://github.com/vuejs/core/blob/main/CHANGELOG.md

Reply:
I used `yarn upgradeInteractive`. 
`3.4.19` fulfills `^3.3.9` so it should be valid.

Reply:
Yet the version change would just be cosmetic ^^

Reply:
Done then :slightly_smiling_face: